spec/controllers/follows_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe FollowsController, type: :controller do
+
+end
```

Comment:
So here's a placeholder controller test for the `FollowsController`, but there's no placeholder unit test for the `Follow` model itself? :thinking: